go.mod

```diff
@@ -2,4 +2,11 @@ module github.com/matrix-org/naffka
 
 go 1.13
 
-require github.com/Shopify/sarama v1.26.1
+require (
+	github.com/Shopify/sarama v1.26.1
+	github.com/mattn/go-sqlite3 v2.0.2+incompatible
+	go.uber.org/atomic v1.4.0
```

Comment:
Please can we use the stdlib for atomic variables to reduce our dependency count? It's literally just syntactic sugar from `sync/atomic` - https://github.com/uber-go/atomic/blob/v1.6.0/atomic.go#L185

Reply:
Looking at `uber.go/atomic`'s deps actually are just stdlib so this is less important: https://godoc.org/go.uber.org/atomic?imports